unlock_account.go

```diff
@@ -35,3 +35,7 @@ func (r *AccountUnlock) Type() UnlockType {
 func (r *AccountUnlock) Size() int {
 	return AccountUnlockSize
 }
+
+func (r *AccountUnlock) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return workScoreStructure.WorkScoreEd25519Signature
```

Comment:
Add the mentioned Signature.WorkScore() function here.

Reply:
Btw, we never charge for these right? Because we never analyse inputs anyway? Maybe we should switch to charge costs if someone creates these things on the output side.

Reply:
I think this one shoudl not actually incur a signature cost because it will already be accounted for by the signature unlock of the account output.

Reply:
Does that sound correct? Same for other reference unlocks...

Reply:
correct